Environment/AirQualityObserved/harvest/madrid_air_quality.py

```diff
@@ -219,8 +219,9 @@ def get_air_quality_madrid(target_stations, target_hour=-1):
                         'streetAddress': station_dict[station_num]['address']
                     },
                     'location': station_dict[station_num]['location'] or None,
-                    'source': 'http://datos.madrid.es',
-                    'dateCreated': datetime.datetime.now().isoformat()
+                    'source': 'http://datos.madrid.es'
+                    # According to read-only nature of dateCreated, it must not be added
+                    #'dateCreated': datetime.datetime.now().isoformat()
```

Comment:
Not sure if this would break anything, I don't know the details of the madrid_air_quality.py script... However, it seems to be pretty evident that dateCreated shouldn't be added here.

Reply:
Maybe the same affect to other similar scripts in the repository (I have seen some other ocurrences of dateCreated there). However, other cases are not so evident and I have preferred not touching without guidance.